content/en/serverless/azure_app_services/azure_app_services_linux.md

```diff
@@ -14,78 +15,133 @@ This instrumentation method provides the following additional monitoring capabil
-{{% tab "Node, .NET, PHP, Python" %}}
-Go to **General settings** and add the following to the **Startup Command** field:
+```ssh
+#!/usr/bin/env bash
 
-```
-curl -s https://raw.githubusercontent.com/DataDog/datadog-aas-linux/v1.12.0/datadog_wrapper | bash
+echo "Setting up Datadog tracing for PHP"
+DD_PHP_TRACER_VERSION=1.8.3
+
+    DD_PHP_TRACER_URL=https://github.com/DataDog/dd-trace-php/releases/download/${DD_PHP_TRACER_VERSION}/datadog-setup.php
+
+echo "Installing PHP tracer from ${DD_PHP_TRACER_URL}"
+if curl -LO --fail "${DD_PHP_TRACER_URL}"; then
+eval "php datadog-setup.php --php-bin=all"
+else
+       echo "Downloading the tracer was unsuccessful"
+       return
+fi
+
+cp /home/site/wwwroot/default /etc/nginx/sites-available/default && service nginx reload
 ```
 
-{{< img src="serverless/azure_app_service/startup-command-1.jpeg" alt="Azure App Service Configuration: the Stack settings, under the Configuration section of Settings in the Azure UI. Underneath the stack, major version, and minor version fields is a 'Startup Command' field that is populated by the above curl command." style="width:100%;" >}}
+This script is intended to run as the startup command, which installs the tracing module into PHP and then restarts the application. 
+
 {{% /tab %}}
-{{% tab "Java" %}}
-Download the [`datadog_wrapper`][8] file from the releases and upload it to your application with the Azure CLI command:
+{{% tab "Python" %}}
 
-```
-  az webapp deploy --resource-group <group-name> --name <app-name> --src-path <path-to-datadog-wrapper> --type=startup
-```
+1. Add `ddtrace` to your project.
+1. Modify your startup command. Your new command should run `ddtrace-run` with your old command as an argument. That is: if your startup command is `foo`, modify it to run `ddtrace-run foo`.
+
+   For example:
+   ```ssh
+   ddtrace-run gunicorn --bind=0.0.0.0 --timeout 600 quickstartproject.wsgi
+   ```
 
-[8]: https://github.com/DataDog/datadog-aas-linux/releases
 {{% /tab %}}
 {{< /tabs >}}
 
-### Viewing traces
+2. **Configure environment variables**.
+   In Azure, add the following key-value pairs in **Settings** > **Configuration** > **Application settings**:
 
-When new Application Settings are saved, Azure restarts the application. However, if a startup command is added and saved, a restart may be required.
+| Name | Value | Description |
+|------|-------|-------------|
+| `DD_API_KEY` | Your Datadog API key. | See [Organization Settings > API Keys][16] in Datadog. |
+| `DD_SITE` | {{< region-param key="dd_site" code="true" >}} | Your [Datadog site][2]. Defaults to `datadoghq.com`. |
+| `DD_SERVICE` | Your application's service name. | Defaults to the name field value in `package.json`. |
 
-After the application restarts, you can view traces by searching for the service name (`DD_SERVICE`) in the [APM Service page][4] of Datadog.
+{{% collapse-content title=".NET: Additional required environment variables" level="h4" id="dotnet-additional-settings" %}}
```

Comment:
Would it be possible for this to appear when the `.NET` tab is selected? It would make it more clear what all of the instructions are for .NET without needing to make multiple clicks.